stories/Templates/CountryHomepage/CountryHomepage.jsx

```diff
@@ -104,6 +106,9 @@ export const CountryHomepage = ({
               data={featureddataafterwedo.contentdata }
               storyBtn={featureddataafterwedo.storyBtn }
             />
+            <div className="cell cta-container">
+                <CtaButton label={storyBtn} For_Primary='No Arrow' />
```

Comment:
@neelesh-tippe Change the text here. Avoid using underscore for inline attributes `For_Primary`.

Reply:
CTA button is imported. This change need to be do on base component.